app/src/main/java/org/wikipedia/views/ViewUtil.java

```diff
@@ -55,14 +56,23 @@ public static void loadImage(@NonNull ImageView view, @Nullable String url, bool
     }
 
     public static void loadImageWithWhiteBackground(@NonNull ImageView view, @Nullable String url) {
+        loadImageWithWhiteBackground(view, url, null);
+    }
+
+    public static void loadImageWithWhiteBackground(@NonNull ImageView view, @Nullable String url, @Nullable RequestListener<Drawable> listener) {
```

Comment:
I see a "blink" when the image being loaded, is it possible to reduce the issue?

Reply:
should be improved now.